src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,94 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_GOODBYE;
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.List;
+
+public class Formatter {
+
+	/**
+	 * A decorative prefix added to the beginning of lines printed by
+	 * AddressBook
+	 */
+	private static final String LINE_PREFIX = "|| ";
+
+	/** A platform independent line separator. */
+	private static final String LS = System.lineSeparator();
+
+	private static final String DIVIDER = "===================================================";
+
+	/** Format of indexed list item */
+	private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+	/** Offset required to convert between 1-indexing and 0-indexing. */
+	public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+	/**
+	 * Format of a comment input line. Comment lines are silently consumed when
+	 * reading user input.
+	 */
+	private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+
+	/** Format of user command prompt. */
+	public static String getUserCommandFormat() {
+		return LINE_PREFIX + "Enter command: ";
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.